src/griffe/loader.py

```diff
@@ -335,6 +337,34 @@ def _store_and_return(self, name: str, module: Module) -> Module:
         self.modules_collection[module.path] = module
         return self.modules_collection[name]  # type: ignore[index]
 
+    def _load_package(self, pkg: Package, submodules: bool = True) -> Module:
+        """Load top module in a package.
```

Comment:
not sure if you want this docstring there or not.  Can remove to keep consistent with other privates here?

Reply:
Yeah if it's private you can remove the docstring :smile: